src/components/grid-view/GridViewSlider.scss

```diff
@@ -0,0 +1,103 @@
+@import '../../elements/common/variables';
+@import '../../styles/variables';
+
+.bdl-GridViewSlider {
+    display: flex;
+    margin-right: 5px;
+}
+
+@mixin bdl-GridViewSlider-Track {
+    background: $bdl-gray-10;
+    border: none;
+    border-radius: 2px;
+    height: 2px;
+}
+
+@mixin bdl-GridViewSlider-Thumb {
+    background: $bdl-gray-80;
+    border: none;
+    border-radius: 50%;
+    height: 10px;
+    width: 10px;
+}
+
+.bdl-GridViewSlider-Range {
+    appearance: none;
+
+    /* fix for FF unable to apply focus style bug */
+    cursor: pointer;
+    margin-left: 7px;
+    margin-right: 7px;
+
+    &::-webkit-slider-runnable-track {
+        @include bdl-GridViewSlider-Track;
+    }
+
+    &::-webkit-slider-thumb {
+        @include bdl-GridViewSlider-Thumb;
+
+        appearance: none;
+        margin-top: -4px;
+        width: 10px;
+    }
+
+    &::-moz-range-track {
+        @include bdl-GridViewSlider-Track;
+    }
+
+    &::-moz-range-thumb {
+        @include bdl-GridViewSlider-Thumb;
+    }
+
+    &::-ms-track {
+        @include bdl-GridViewSlider-Track;
+
+        color: transparent;
+    }
+
+    &::-ms-fill-lower {
+        background: $bdl-gray-10;
+        border-radius: 10px;
+    }
+
+    &::-ms-fill-upper {
+        background: $bdl-gray-10;
+        border-radius: 10px;
+    }
+
+    &::-ms-thumb {
+        @include bdl-GridViewSlider-Thumb;
+    }
+
+    /* hide the outline behind the border */
+    &:-moz-focusring {
+        outline: 1px solid $white;
+        outline-offset: -1px;
+    }
+
+    &:focus {
+        outline: none;
+    }
+
+    &:focus::-ms-fill-lower {
+        background: $bdl-gray-50;
+    }
+
+    &:focus::-ms-fill-upper {
+        background: $bdl-gray-20;
+    }
+}
+
+.bdl-GridViewSlider-PlainButton {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+
+    &.bdl-GridViewSlider-PlainButton--minus {
```

Comment:
This can live at the top level, as the extra specificity shouldn't be necessary if it appears below the class it's overriding.